services/workflows-service/src/workflow/workflow.service.ts

```diff
@@ -672,9 +672,11 @@ export class WorkflowService {
     {
       workflowId,
       documentId,
+      documentsUpdateContextMethod,
     }: {
       workflowId: string;
       documentId: string;
+      documentsUpdateContextMethod?: 'base' | 'director';
```

Comment:
Might want to export to an enum or at least a type.

Reply:
In my TODO